surveys/2022-annual-survey/questions.md

```diff
@@ -26,41 +26,12 @@ Type: select one
 
 ## For previous Rust users
 
-### As you have indicated that you're no longer using Rust, what prompted you to participate in this survey?
-
-Type: select all that apply (optional)
-
-- I plan to return to using Rust in the future
-- I consider myself part of the Rust community
-- Specifically to provide feedback on why I stopped using Rust
-- To provide feedback on Rust in general
-- Curiosity
-- Other (open response)
-
-> **justification**
->
-> Useful in understanding why non-users contribute;
-
 > **SURVEY FLOW**
 >
 > Skip to `## Your opinions about Rust` section
 
 ## For non-Rust users
 
-### As you have indicated that you have not used Rust, what prompted you to participate in this survey?
```

Comment:
Perhaps we want to replace these with a TODO just so we're sure we don't forget that we should either make new questions here or delete these sections entirely.

Reply:
I have the TODOs in a separate branch